tests/src/rules/no-unlocalized-strings.test.ts

```diff
@@ -198,6 +198,10 @@ ruleTester.run<string, Option[]>(name, rule, {
       code: 'class Form extends Component { displayName = "FormContainer" };',
       options: [{ ignoreNames: ['displayName'] }],
     },
+    {
+      code: 'const Shape = { CIRCLE: "circle" as const };',
+      options: [{ ignoreNames: [{ regex: { pattern: '^[A-Z0-9_-]+$' } }] }],
+    },
```

Comment:
let's add a name to the test case, this will be easier in future to understand what we checking here